server/validation.js

```diff
@@ -0,0 +1,104 @@
+/*
+question.js
```

Comment:
rename to questionValidator (the file as well). we are going to need a user validator later on.